source/_integrations/tado.markdown

```diff
@@ -38,6 +38,8 @@ There is currently support for the following device types within Home Assistant:
 - Sensor - for some additional information of the zones.
 - Weather - for information about the current weather at the location of your Tado home.
 
```

Comment:
```suggestion
## Unsupported device types

```
I think this note is worth its own header. Could you also add a title ## Supported device types in line 31? I would make a suggestion but it too far out so I'm not allowed.